packages/api/src/controllers/v1/agit/mod.rs

```diff
@@ -1,95 +1,197 @@
-        Path(AgitPathParam { id }): Path<AgitPathParam>,
-    ) -> Result<Json<Agit>> {
-        tracing::debug!("get agit {id}");
-        Ok(Json(Agit::default()))
+
+    pub fn route(pool: sqlx::PgPool) -> Result<by_axum::axum::Router> {
+        let ctrl = Self::new(pool);
+        Ok(by_axum::axum::Router::new()
+            .route("/:id", get(Self::get_agit_by_id).post(Self::act_agit_by_id))
+            .route("/", post(Self::act_agit).get(Self::get_agit))
+            .with_state(ctrl))
     }
-    pub async fn act(
+
+    pub async fn act_agit(
         State(ctrl): State<AgitControllerV1>,
-        Extension(claim): Extension<Option<Authorization>>,
+        Extension(auth): Extension<Option<Authorization>>,
         Json(body): Json<AgitAction>,
     ) -> Result<Json<Agit>> {
-        tracing::debug!("agit act {body:?}");
+        tracing::debug!("act_agit {:?}", body);
+
         match body {
-            AgitAction::Create(req) => {
-                //TODO: Add Create Agit
-                Ok(Json(Agit::default()))
+            AgitAction::Create(param) => {
+                let res = ctrl.create(auth, param).await?;
+                Ok(Json(res))
             }
         }
     }
 
-    pub async fn act_by_id(
+    pub async fn act_agit_by_id(
         State(ctrl): State<AgitControllerV1>,
-        Path(AgitPathParam { id }): Path<AgitPathParam>,
-        Extension(claim): Extension<Option<Authorization>>,
+        Extension(auth): Extension<Option<Authorization>>,
+        Path(AgitPath { agit_id }): Path<AgitPath>,
         Json(body): Json<AgitByIdAction>,
     ) -> Result<Json<Agit>> {
-        tracing::debug!("agit act_by_id {id} {body:?}");
+        tracing::debug!("act_agit_by_id {} {:?}", agit_id, body);
+
         match body {
-            AgitByIdAction::Update(_) => {
-                //TODO: Add Update Agit
-                Ok(Json(Agit::default()))
+            AgitByIdAction::Update(param) => {
+                let res = ctrl.update(agit_id, auth, param).await?;
+                Ok(Json(res))
             }
             AgitByIdAction::Delete(_) => {
-                //TODO: Add Delete Agit
-                Ok(Json(Agit::default()))
+                let res = ctrl.delete(agit_id, auth).await?;
+                Ok(Json(res))
             }
         }
     }
+
+    pub async fn get_agit_by_id(
+        State(ctrl): State<AgitControllerV1>,
+        Extension(_auth): Extension<Option<Authorization>>,
+        Path(AgitPath { agit_id }): Path<AgitPath>,
+    ) -> Result<Json<Agit>> {
+        tracing::debug!("get_agit {}", agit_id);
+        Ok(Json(
+            Agit::query_builder()
+                .id_equals(agit_id)
+                .query()
+                .map(Agit::from)
+                .fetch_one(&ctrl.pool)
+                .await?,
+        ))
+    }
+
+    pub async fn get_agit(
+        State(ctrl): State<AgitControllerV1>,
+        Path(AgitParentPath { agit_id }): Path<AgitParentPath>,
```

Comment:
it doesn't need Path param.
Refer to https://github.com/biyard/incheon-heroes/blob/dev/packages/main-api/src/controllers/v1/contents.rs#L439

In the case of incheonheroes, the axum router belongs to `/v2/organzation/:org-id` So it needed ParentPath.